Node/src/botbuilder-teams.d.ts

```diff
@@ -1108,6 +1108,15 @@ export class TeamsChatConnector extends builder.ChatConnector {
   */
   public fetchMembers(serviceUrl: string, conversationId: string, callback: (err: Error, result: ChannelAccount[]) => void) : void;
 
+  /**
+  *  Return a list of members in team or chat.
+  *  @param {string} serviceUrl - The service url for the team or chat, which should be taken from a previous message received from that team or chat. If the wrong service url is used, the method wil fail.
+  *  @param {string} conversationId - The team id or chat conversation id.
+  * @param {string} memberId - Member Id
+  *  @param {function} callback - Function to invoke with the list of members.
+  */
+ public fetchMember(serviceUrl: string, conversationId: string, memberId: string,  callback: (err: Error, result: ChannelAccount[]) => void) : void;
```

Comment:
it's supposed to design result as `ChannelAccount` type rather than `ChannelAccount[]`